stories/Customize/CSSVariables.stories.mdx

```diff
@@ -24,111 +24,7 @@ Here are the variables we include that are accessible from any location where ne
-  --neeto-ui-info-500: 20, 115, 230;
-  --neeto-ui-info-100: 226, 242, 255;
-  --neeto-ui-shadow-xs: 0px 1px 4px -1px rgba(28, 48, 74, 0.12);
-  --neeto-ui-shadow-sm: 0px 3px 12px -1px rgba(28, 52, 84, 0.12), 0px 2px
-      4px -1px rgba(28, 55, 90, 0.08);
-  --neeto-ui-shadow-md: 0px 8px 24px -4px rgba(28, 50, 79, 0.12), 0px 2px
-      6px -1px rgba(28, 55, 90, 0.08);
-  --neeto-ui-shadow-lg: 0 8px 24px rgba(140,149,159,0.2);
-  --neeto-ui-text-h1: 32px;
-  --neeto-ui-text-h2: 24px;
-  --neeto-ui-text-h3: 20px;
-  --neeto-ui-text-h4: 16px;
-  --neeto-ui-text-h5: 14px;
-  --neeto-ui-text-h6: 12px;
-  --neeto-ui-text-body1: 16px;
-  --neeto-ui-text-body2: 14px;
-  --neeto-ui-text-body3: 12px;
-  --neeto-ui-text-nano: 10px;
-  --neeto-ui-text-xxs: 10px;
-  --neeto-ui-text-xs: 12px;
-  --neeto-ui-text-sm: 14px;
-  --neeto-ui-text-base: 16px;
-  --neeto-ui-text-lg: 18px;
-  --neeto-ui-text-xl: 20px;
-  --neeto-ui-text-2xl: 24px;
-  --neeto-ui-text-3xl: 32px;
-  --neeto-ui-text-4xl: 48px;
-  --neeto-ui-font-thin: 100;
-  --neeto-ui-font-extralight: 200;
-  --neeto-ui-font-light: 300;
-  --neeto-ui-font-normal: 400;
-  --neeto-ui-font-medium: 500;
-  --neeto-ui-font-semibold: 600;
-  --neeto-ui-font-bold: 700;
-  --neeto-ui-font-extrabold: 800;
-  --neeto-ui-font-black: 900;
-  --neeto-ui-leading-none: 1;
-  --neeto-ui-leading-tight: 1.25;
-  --neeto-ui-leading-snug: 1.375;
-  --neeto-ui-leading-normal: 1.5;
-  --neeto-ui-leading-relaxed: 1.625;
-  --neeto-ui-leading-loose: 2;
-  --neeto-ui-text-transform-none: none;
-  --neeto-ui-text-transform-capitalize: capitalize;
-  --neeto-ui-text-transform-uppercase: uppercase;
-  --neeto-ui-text-transform-lowercase: lowercase;
-  --neeto-ui-text-transform-full-width: full-width;
-  --neeto-ui-text-transform-inherit: inherit;
-  --neeto-ui-text-transform-initial: initial;
-  --neeto-ui-text-transform-revert: revert;
-  --neeto-ui-text-transform-unset: unset;
-  --neeto-ui-rounded-none: 0;
-  --neeto-ui-rounded-sm: 3px;
-  --neeto-ui-rounded: 5px;
-  --neeto-ui-rounded-md: 6px;
-  --neeto-ui-rounded-lg: 8px;
-  --neeto-ui-rounded-xl: 12px;
-  --neeto-ui-rounded-full: 999px;
-  --neeto-ui-transition: all 0.3s linear;
-  --neeto-ui-modal-z-index: 99999;
-  --neeto-ui-pane-header-height: 78px;
-  --neeto-ui-pane-footer-height: 80px;
-  --neeto-ui-main-header-height: 80px;
-  --neeto-ui-main-header-with-breadcrumbs-height: 94px;
-  --neeto-ui-sub-header-height: 37px;
-  --neeto-ui-sub-header-bottom-margin: 20px;
-  --neeto-ui-pagination-block-height: 88px;
-  --neeto-ui-table-header-height: 40px;
-  --neeto-ui-surface-bg: 255, 255, 255;
-  --neeto-ui-pastel-color-1: 229, 231, 235;
-  --neeto-ui-pastel-color-2: 254, 202, 202;
-  --neeto-ui-pastel-color-3: 253, 230, 138;
-  --neeto-ui-pastel-color-4: 167, 243, 208;
-  --neeto-ui-pastel-color-5: 191, 219, 254;
-  --neeto-ui-pastel-color-6: 199, 210, 254;
-  --neeto-ui-pastel-color-7: 221, 214, 254;
-  --neeto-ui-pastel-color-8: 251, 207, 232;
+
```

Comment:
Shouldn't include the variables here?

Reply:
The component-specific variables are listed at the bottom of the respective components with an override example.

Example:

<img width="525" alt="Screenshot 2023-12-01 at 11 02 40 PM" src="https://github.com/bigbinary/neeto-ui/assets/24496302/d5442e8f-a628-4c76-a81b-a4e4293644ba">